.github/workflow/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Erlang CI
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "sbom.json"
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - "sbom.json"
+
+jobs:
+  ct_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        otp: ['27', '28']
+        rebar3: ['3.25.1']
+  steps:
```

Comment:
`steps` is not inside `ct-tests`, probably this makes the workflow un-runnable, check the indentations